### ISSUE TITLE
Add settings

### ISSUE THREAD

**Issue by GijsGoudzwaard** [GijsGoudzwaard/Image-Optimizer#14]
Add a settings page, where the user can specify how good or bad they want to compress their images.

**Comment by stiffler17** [comment]
Hi!

I like the interface of https://kraken.io/web-interface
Maybe you'll find it as good as I do.
It is very clear but nevertheless full of settings.

**Comment by aries223** [comment]
Yes I second the Settings. Add the ability to set the amount of compression we desire. This gives us the ability to compress the file enough to meet file size requirements..
(take a look at ImageOptim on the Mac)
